js/js-modern-syntax_rest-and-spread/index.js

```diff
@@ -49,3 +52,4 @@ export const result = add(numbers);
 
 const fruits = ["apple", "banana", "orange", "papaya"];
 const anotherFruit = "cherry";
+export const allFruits = [anotherFruit, fruits];
```

Comment:
I think it should be: `export const allFruits = [anotherFruit, ...fruits];

=> without the ... it returns an array within an array: 
`[
  'cherry',
  [ 'apple', 'banana', 'orange', 'papaya' ]
]`